### PR TITLE
Fixed zoom for view with axes

### PR DESCRIPTION
**Pull request by shadab-skhan**

### CODE REVIEW

File: react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx [hunk 1]
```diff
@@ -403,7 +403,7 @@ export const Axes = MinimalTemplate.bind({});
 Axes.args = {
     id: "axes",
     layers: [axes, meshMapLayer, north_arrow_layer],
-    bounds: [432150, 6475800, 439400, 6481500],
+    bounds: [432205, 6475078, 437720, 6481113],
```
Comment (shadab-skhan) [inline react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx:406]: This bound should be same as axis bounds.
Ideally, project bounds should be calculated based on data bounds of all the layers.

**Comment by hkfb** [comment]
What is the problem we are solving with this? That the initial viewport is too small when the axes are shown?

**Comment by shadab-skhan** [comment]
Before:
image

After: 
image

This change only corrects the bounds of storybook example for axes story.

File: react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx (hunk 1 above)
Comment (hkfb) [reply react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx:406]: We have an issue for automatically calculating the bounds (https://github.com/equinor/webviz-subsurface-components/issues/924), but until we have that in place, then I think using the bounds itself to set the initial viewport is the best solution.

**Review by hkfb — CHANGES_REQUESTED** [review]

**Comment by shadab-skhan** [comment]
Discarding PR

**Pull request closed without merging**